jenkins/check_copyright.py

```diff
@@ -42,6 +42,8 @@ def report_incorrect(file_name, pairs):
 linecount = 0
 pairs = []
 with open(file_name, 'r') as sourcefile:
+    if not re.match(r'#!/usr/bin/env.*', sourcefile.readline(), 0):
+        sourcefile.seek(0)
```

Comment:
Changes will still be code reviewed and we can catch that sort of thing. In fact, there's nothing stopping someone from adding a comment at the end of the first shebang line to do the same no matter what structure you impose here. You don't know for a fact where the binaries will live and making assumptions like that seems bad.